src/utils/__tests__/getFlowType-test.js

```diff
@@ -232,6 +232,39 @@ describe('getFlowType', () => {
     }, raw: '{ a: string, b: ?xyz }'});
   });
 
+  it('handles ObjectTypeSpreadProperty', () => {
+    var typePath = statement(`
+      var x: { ...OtherProps, ...MyType, a: string };
+
+      type MyType = { myString: string };
+    `).get('declarations', 0).get('id').get('typeAnnotation').get('typeAnnotation');
+
+    expect(getFlowType(typePath)).toEqual({
+      name: 'signature',
+      type: 'object',
+      signature: {
+        inheritedProperties: [
```

Comment:
importedProperties

Reply:
or compose like in the flow PR? From a purely react perspective, I think `composes` makes a lot of sense. When describing the props it could (very unlikely) also be a global that gets spread into the props, where this name then does not make sense. (Btw we should add a test for spreading a global variable, which has no declaration in the scope)